src/lib/signature_lib/checked.ml

```diff
@@ -55,9 +66,9 @@ module type S = sig
      and type ('a, 'b) checked := ('a, 'b) checked
 
   module Signature : sig
-    type t = curve_scalar * curve_scalar [@@deriving sexp]
+    type t = bool list * curve_scalar [@@deriving sexp]
```

Comment:
we could also use field elements throughout if that's preferable to bool lists for (x, y) ?

Reply:
this is still a bool list, should it be changed?